setup/docker/base/Dockerfile

```diff
@@ -18,7 +18,19 @@ RUN \
     mkdir -p /usr/share/illarion/scripts && \
     mkdir /scripts && \
     mkdir /maps && \
-    ln -s /maps /usr/share/illarion/map/import
+    ln -s /maps /usr/share/illarion/map/import && \
+    # Setup user running the server
+    groupadd --system illarion && \
+    useradd -rm -d /home/illarion --shell /bin/bash -g illarion -G users illarion && \
+    chown :illarion /usr/share/illarion/map && \
+    chown :illarion /usr/share/illarion/scripts && \
+    chmod -R a+r /scripts /maps && \
+    chmod -R g+w /usr/share/illarion/scripts /usr/share/illarion/map && \
+    chmod a+r /etc/illarion.conf && \
+    chmod a+rx /usr/share/illarion/pre-reload
+
+USER illarion
+WORKDIR /home/illarion
```

Comment:
I think we can remove the custom user altogether. We can grant write access for everyone to `/usr/share/illarion/map` and `/usr/share/illarion/scripts` and then set the user via docker-compose to run as e.g. devserver.
In this scenario the executing user on the host would be responsible for running the container in a secure way, which sounds fine.

Reply:
Done that.